.github/workflows/Web.yaml

```diff
@@ -0,0 +1,44 @@
+name: Static Web Apps
```

Comment:
Please remove stage for deploying to GP, for test environment you should have a docker image first. Also please add a linter stage and stage for tests.